README.md

```diff
@@ -446,3 +442,22 @@ requests
 | where customDimensions.clientId == "prisoner-offender-search-client"
 | order by timestamp desc ```
 ```
+<br />
+<br />
+<br />
+<br />
+<br />
+
+
+
+
+——
+<br />
+
+<sup><sup><sup><em>1</em></sup><em>&nbsp;Before&nbsp;running&nbsp;locally&nbsp;when&nbsp;using&nbsp;Docker&nbsp;Desktop&nbsp;on&nbsp;Windows,&nbsp;do&nbsp;&nbsp;"<code>dos2unix&nbsp;setup-xxx.sh</code>"&nbsp;in&nbsp;Git&nbsp;Bash.&nbsp;That&nbsp;prevents&nbsp;the&nbsp;"<code>invalid&nbsp;optionnt-initaws.d/setup-es.sh:&nbsp;line&nbsp;2:&nbsp;set:&nbsp;-</code>"&nbsp;error&nbsp;you&nbsp;would&nbsp;get&nbsp;otherwise.</em></sup></sup>
```

Comment:
this section needs formatting properly - remove `nbsp`.  Worth also clicking _View file_ since the text is too small and unreadable

Reply:
also should be markdown rather than html

Reply:
also shouldn't really have to run `dos2unix`, ideally we should set `.gitattributes` appropriately.  See https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings for more information

Reply:
wondering whether these footnotes would actually be better in the document rather than hidden away here.  Since github doesn't support footnotes the reader would have to go up and down the document and the contents offers valuable advice for windows users.

Reply:
> "..._also should be markdown rather than html_..."

It's a well-kept secret. There are a handful of HTML tags that are appropriate in Markdown...

>>   ...
>> 
>> #### [_Basic Syntax | Markdown Guide_](https://www.markdownguide.org/basic-syntax)
>> ...
>>       
>> "..._Fortunately, there is another option supported by nearly every Markdown application: ...HTML_..."

Reply:
> "_...clicking View file..._"

Whenever I need to edit _`*.md`_ files (_or minor edits to other files; like ones in this pull request_), I'm in the habit of always using a relatively more lightweight IDE called [_Visual Studio Code_](https://code.visualstudio.com). The Markdown preview key combination (_on Windows_) is _`CTRL + SHIFT + V`_ .

That detail doesn't matter, of course. But I'm so impressed with vscode, I take any and every opportunity to plug it whenever one arises :¬)

Reply:
I know that HTML is supported in markdown.  My point was that we use markdown instead of html in our files as it then makes it easier for everyone to edit and parse - this is especially important when resolving conflicts.